src/lib/public/board/data/officers.json

```diff
@@ -1,718 +1,368 @@
+    "picture": "mike-ploythai.webp",
     "positions": {
-      "S21": {
-        "title": "Create Officer",
-        "tier": 17
-      },
-      "F21": {
-        "title": "Create Director, Marketing Chair",
-        "tier": 16
-      },
-      "S22": {
-        "title": "Create President",
-        "tier": 16
-      }
-    },
-    "picture": "mike-ploythai.webp"
+      "S21": { "title": "Create Officer", "tier": 16 },
+      "F21": { "title": "Create Director, Marketing Chair", "tier": 13 },
+      "S22": { "title": "Create President", "tier": 13 }
+    }
   },
   {
     "fullName": "Stephanie Pocci",
+    "picture": "stephanie-pocci.webp",
     "positions": {
-      "S22": {
-        "title": "Event Coordinator",
-        "tier": 9
-      },
-      "F22": {
-        "title": "Event Coordinator",
-        "tier": 9
-      }
-    },
-    "picture": "stephanie-pocci.webp"
+      "S22": { "title": "Event Coordinator", "tier": 10 },
+      "F22": { "title": "Event Coordinator", "tier": 10 }
+    }
   },
   {
     "fullName": "Nicolas Renteria",
-    "positions": {
-      "S21": {
-        "title": "Marketing Chair",
-        "tier": 12
-      }
-    },
-    "picture": "nicolas-renteria.webp"
+    "picture": "nicolas-renteria.webp",
+    "positions": { "S21": { "title": "Marketing Chair", "tier": 8 } }
   },
   {
     "fullName": "Wilbert Rodriguez",
-    "positions": {
-      "S21": {
-        "title": "Intern Program Manager",
-        "tier": 11
-      }
-    },
-    "picture": "wilbert-rodriguez.webp"
+    "picture": "wilbert-rodriguez.webp",
+    "positions": { "S21": { "title": "Intern Program Manager", "tier": 22 } }
   },
   {
     "fullName": "Samuel Sandoval",
-    "positions": {
-      "S21": {
-        "title": "Vice President, Dev Director",
-        "tier": 1
-      }
-    },
-    "picture": "samuel-sandoval.webp"
+    "picture": "samuel-sandoval.webp",
+    "positions": { "S21": { "title": "Vice President, Dev Director", "tier": 1 } }
   },
   {
     "fullName": "Kavit Sanghavi",
-    "positions": {
-      "S21": {
-        "title": "Algo Director",
-        "tier": 11
-      }
-    },
-    "picture": "kavit-sanghavi.webp"
+    "picture": "kavit-sanghavi.webp",
+    "positions": { "S21": { "title": "Algo Director", "tier": 11 } }
   },
   {
     "fullName": "Parth Sharma",
-    "positions": {
-      "S21": {
-        "title": "Algo Officer",
-        "tier": 15
-      }
-    },
-    "picture": "parth-sharma.webp"
+    "picture": "parth-sharma.webp",
+    "positions": { "S21": { "title": "Algo Officer", "tier": 12 } }
   },
   {
     "fullName": "Justin Stitt",
+    "picture": "justin-stitt.webp",
     "positions": {
-      "S22": {
-        "title": "Algo Officer",
-        "tier": 15
-      },
-      "F22": {
-        "title": "AI Team Lead",
-        "tier": 12
-      }
-    },
-    "picture": "justin-stitt.webp"
+      "S22": { "title": "Algo Officer", "tier": 12 },
+      "F22": { "title": "AI Team Lead", "tier": 11 }
```

Comment:
The updated "tier" for AI Lead is '20'

`"AI Leader": { "id": 20, "index": 1100 },`